nuxeo-ai-config/src/test/java/org/nuxeo/ai/TestAIConfigREST.java

```diff
@@ -104,15 +106,16 @@ public void iCanSetNuxeoConfVar() throws IOException {
         Map<String, String> confVar = new HashMap<>();
         confVar.put("test", "some value");
         String body = mapper.writeValueAsString(confVar);
-        try (CloseableClientResponse response = getResponse(BaseTest.RequestType.POST, "ai/config", body)) {
+        try (CloseableClientResponse response = getResponse(BaseTest.RequestType.POST, "aicore/config", body)) {
             assertThat(response.getStatus()).isEqualTo(Response.Status.OK.getStatusCode());
             assertThat(Framework.getProperty("test")).isNotNull().isNotEmpty();
         }
     }
 
     @Test
     public void iCanSetThreshold() {
-        try (CloseableClientResponse response = getResponse(RequestType.POST, "ai/extension/thresholds", thresholdFile,
+        try (CloseableClientResponse response = getResponse(RequestType.POST,
+                "aicore/extension/thresholds?docType=File", thresholdFile,
```

Comment:
The test fails https://jenkins.ai.dev.nuxeo.com/blue/organizations/jenkins/nuxeo%2Fnuxeo-ai/detail/PR-272/5/tests